Source/WebCore/rendering/RenderText.cpp

```diff
@@ -1336,7 +1336,7 @@ void RenderText::computePreferredLogicalWidths(float leadWidth, WeakHashSet<cons
     if (!style.autoWrap())
         m_minWidth = m_maxWidth;
 
-    if (style.whiteSpaceCollapse() == WhiteSpaceCollapse::Preserve && style.textWrap() == TextWrap::NoWrap) {
+    if (style.whiteSpaceCollapse() == WhiteSpaceCollapse::Preserve && style.textWrapMode() == TextWrapMode::NoWrap) {
```

Comment:
I really wish we had a boolean for `text-wrap-mode` instead of an enumeration for call sites like this.

Reply:
ditto